build/templates/_converters.py.mako

```diff
@@ -153,6 +153,59 @@ def convert_repeated_capabilities_without_prefix(repeated_capability):
     return ','.join(convert_repeated_capabilities(repeated_capability, ''))
 
 
+def convert_single_group_repeated_capabilities(repeated_capability):
+    '''Convert a repeated capabilities string, with at most one prefix (that ends with '/') and without any comma, to a list
+
+    Args:
+        repeated_capability (str) - Supported types (with an optional prefix):
+            - range (using '-' or ':')
+            - single item
+
+    Returns:
+        rep_cap (str) - comma delimited list of strings of the expanded repeated capability items
+    '''
+    if '/' in repeated_capability:
+        split_index = repeated_capability.find('/') + 1
+        prefix = repeated_capability[:split_index]
+        return convert_repeated_capabilities(repeated_capability[split_index:], prefix)
+    return convert_repeated_capabilities(repeated_capability)
+
+
+def convert_channels_repeated_capabilities(
```

Comment:
> I think we have bigger problems than the function signature being weird. The below code snippet should tell you the issue:

I believe this is not an issue as the converter function works on "channels repeated capabilities" (as suggested by the function name) instead of "repeated capabilities" in general. That means, if there are multiple instruments in the session, the "channels repeated capabilities" must be fully-qualified, otherwise the driver runtime would throw an error:
```
>>> with nidcpower.Session(resource_name="dev1,dev2", channels="", options="Simulate=1, DriverSetup=Model:4147; BoardType:PXIe", independent_channels=True) as smu:
...     with smu.initiate():
...             smu.channels[6].measure_multiple()
...
nidcpower.errors.DriverError: -1074135008: IVI: Unknown channel or repeated capability name.

Channel Name: 6
```

That also implies that if the "channels repeated capabilities" has no instrument prefix and it still works, the session has only one instrument in it, and `get_channel_names()` would return the same instrument prefix (if it does return an instrument prefix) regardless of which channel it is called with.

Reply:
Thanks for pointing it out. I believe that bug doesn't affect my implementation as it involves `independent_channels = False`, which means `first_channel_name` will not include an instrument prefix and thus there will not be a need to add any instrument info to the final list of channel names.